Fix:
qtpng.c:74:5: error: implicitly declaring library function 'memcpy' with type 'void *(void *, const void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
qtpng.c:213:7: error: implicitly declaring library function 'strcasecmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
https://sourceforge.net/p/libquicktime/git/ci/55f5d4523a96cdf2a45a3af81b8d251337c5d020/
--- plugins/png/qtpng.c.orig	2011-01-07 08:40:46.000000000 -0600
+++ plugins/png/qtpng.c	2021-07-27 15:24:50.000000000 -0500
@@ -25,6 +25,8 @@
 #include "lqt_private.h"
 #include <quicktime/colormodels.h>
 #include <png.h>
+#include <string.h>
+#include <strings.h>
 #include <stdlib.h>
 #include "qtpng.h"
 
